doc/how_to_guides/templates.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Templates
+description:
+type: how_to_guide
+created_at: 2024-11-21
+---
+
+# Templates
+
+## Overview
+As you should know from our documentation their [usage](/usage.md),
+a onesie is already a sort of template. This guide will discuss the
+more specifically customized templates that allow us to define and
+easily re-use elements common among tasks of various generic types.
+
+## Why use templates?
+Templates generally consist of taking the same actions as previous
+onesies and performing them on new sets of arguments. They can
+make it easier for others to review your code by separating out what
+we've done before and what is new about the current effort. In a
+broader sense, they help us all avoid the tedium and error risk that
+can occur when digging up, identifying, and ultimately copying similar
+code from the past and adapting it for the needs of today.
+
```

Comment:
Just my opinion, but I find this not very helpful. I appreciate the converational tone, but I think we can assume devs know the value of templates; when I'm reading documentation I want to get to the meat as quickly as possible.

```suggestion

```

Reply:
I guess I figure a well-organized document can be skimmed according to appetite. The one thing _no_ reader can do is read _more_ than we include.

Reply:
Anyway I axed the why and pruned the overview.